internal/export/database/export_test.go

```diff
@@ -588,3 +588,5 @@ func TestAddExportFileSkipsDuplicates(t *testing.T) {
 		t.Fatalf("bucket name mismatch got %q, want %q", got.BucketName, wantBucketName)
 	}
 }
+
+// TODO TestDeleteFilesBefore
```

Comment:
this should either reference an issue or a user

TODO(#427)
or
TODO(username)

or - just revert this file if the note is just for yourself.

Reply:
I mentioned my username. I think TODO can be resolved in separate PR though it is related to same issue.